### PR TITLE
Attach the APIs library panel to the maven wizard.

### PR DESCRIPTION
**Pull request by nbashirbello**
Subsequent PR will add dependencies.
Part of the fix for #803

### CODE REVIEW

**Comment by nbashirbello** [comment]
@elharo @akerekes

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java [hunk 1]
```diff
@@ -232,6 +236,16 @@ public boolean validatePage() {
     return true;
   }
 
+  public List<Library> getSelectedLibraries() {
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java]: does this need to be public or is it only called from the same package?

Comment (nbashirbello) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java]: I think it should be public

**Comment by elharo** [comment]
Test failures look related:

-------------------------------------------------------
Running com.google.cloud.tools.eclipse.appengine.newproject.maven.MavenArchetypeProjectWizardTest
Tests run: 9, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.32 sec <<< FAILURE! - in com.google.cloud.tools.eclipse.appengine.newproject.maven.MavenArchetypeProjectWizardTest
testAutoPackageNameSetterOnGroupIdChange_whitespaceInGroupId(com.google.cloud.tools.eclipse.appengine.newproject.maven.MavenArchetypeProjectWizardTest)  Time elapsed: 0.548 sec  <<< ERROR!
org.eclipse.core.runtime.AssertionFailedException: null argument:Validation realm cannot be null
	at org.eclipse.core.runtime.Assert.isNotNull(Assert.java:85)
	at org.eclipse.core.databinding.DataBindingContext.<init>(DataBindingContext.java:95)
	at org.eclipse.core.databinding.DataBindingContext.<init>(DataBindingContext.java:82)
	at com.google.cloud.tools.eclipse.appengine.ui.AppEngineLibrariesSelectorGroup.addDatabindingForDependencies(AppEngineLibrariesSelectorGroup.java:108)
	at com.google.cloud.tools.eclipse.appengine.ui.AppEngineLibrariesSelectorGroup.createContents(AppEngineLibrariesSelectorGroup.java:83)
	at com.google.cloud.tools.eclipse.appengine.ui.AppEngineLibrariesSelectorGroup.<init>(AppEngineLibrariesSelectorGroup.java:57)
	at com.google.cloud.tools.eclipse.appengine.newproject.maven.MavenAppEngineStandardWizardPage.createControl(MavenAppEngineStandardWizardPage.java:96)
	at org.eclipse.jface.wizard.Wizard.createPageControls(Wizard.java:175)
	at com.google.cloud.tools.eclipse.appengine.newproject.maven.MavenArchetypeProjectWizardTest.testAutoPackageNameSetterOnGroupIdChange_whitespaceInGroupId(MavenArchetypeProjectWizardTest.java:114)

**Pull request merged**